samples/notesync/NoteSync.html

```diff
@@ -0,0 +1,300 @@
+        document.addEventListener("sync", function() { 
+            that.handleSyncUpdate(event.detail);
+        });
+    },
+
+    handleSyncUpdate: function(sync) {
+        var progress =  (typeof sync.progress === "number" && sync.progress < 100) ? " (" + Math.round(sync.progress) + "%)" : "";
+
+        if (sync.type === "syncDown") {
+            this.lastSoqlSyncDownId = sync.syncId;
+            $(".syncdown").html("Sync " + progress);
+        }
+        if (sync.type === "syncUp") {
+            $(".syncup").html("Sync Up" + progress);
+        }
+        if (sync.status === "DONE") { this.search(); }
+    },
+
+    render: function(eventName) {
+        $(this.el).html(this.template());
+        $(".search-key", this.el).val(this.model.getCriteria());
+        this.listView.setElement($("ul", this.el)).render();
+        return this;
+    },
+
+    syncDown: function() {
+        var target = {type:"custom", androidImpl:"com.salesforce.samples.notesync.ContentSoqlSyncTarget", query:"SELECT " + this.fieldlist.join(",") + " FROM ContentNote LIMIT 1000"};
+        cordova.require("com.salesforce.plugin.smartsync").syncDown(target, "notes", {mergeMode:Force.MERGE_MODE_DOWNLOAD.LEAVE_IF_CHANGED}, this.handleSyncUpdate.bind(this));
+    },
+
+    syncUp: function(event) {
+        cordova.require("com.salesforce.plugin.smartsync").syncUp("notes", {fieldlist: this.fieldlist}, this.handleSyncUpdate.bind(this));
+    },
+
+    inspect: function() {
+        cordova.require("com.salesforce.plugin.smartstore").showInspector();
+    },
+
+    search: function(event) {
+        this.model.setCriteria($(".search-key", this.el).val());
+        this.model.fetch();
+    }
+});
+
+app.views.NoteListView = Backbone.View.extend({
+
+    listItemViews: [],
+
+    initialize: function() {
+        this.model.bind("reset", this.render, this);
+    },
+
+    render: function(eventName) {
+        _.each(this.listItemViews, function(itemView) { itemView.close(); });
+        this.listItemViews = _.map(this.model.models, function(model) { return new app.views.NoteListItemView({model: model}); });
+        $(this.el).append(_.map(this.listItemViews, function(itemView) { return itemView.render().el;} ));
+        return this;
+    }
+
+});
+
+app.views.NoteListItemView = Backbone.View.extend({
+
+    tagName: "li",
+    template: _.template($("#note-list-item").html()),
+
+    render: function(eventName) {
+        $(this.el).addClass("table-view-cell").html(this.template(this.model.toJSON()));
+        return this;
+    },
+
+    close: function() {
+        this.remove();
+        this.off();
+    }
+});
+
+app.views.NotePage = Backbone.View.extend({
+
+    template: _.template($("#note-page").html()),
+
+    events: {
+        "change": "change",
+        "click .save": "save"
+    },
+
+    render: function(eventName) {
+        var templateData = _.extend({ContentDecoded: atob(this.model.get("Content"))}, this.model.toJSON());
```

Comment:
Moving the decoding deeper (e.g. at the time of sync down/up) would have advantages (less space used, and the field could actually be searchable) but would require custom handling during sync down (which we can now do through custom sync target) and during sync up (which we don't have at this point).